bindings/go/Makefile.am

```diff
@@ -10,16 +10,15 @@ CGOCFLAGS=-I$(abs_top_builddir)/src -I$(top_srcdir)/src
 CGOLDFLAGS=-L$(abs_top_builddir)/src/ucp/$(objdir) -lucp -L$(abs_top_builddir)/src/ucs/$(objdir) -lucs
 UCX_SOPATH=$(abs_top_builddir)/src/ucp/$(objdir):$(abs_top_builddir)/src/ucs/$(objdir):$(abs_top_builddir)/src/ucm/$(objdir):$(abs_top_builddir)/src/uct/$(objdir)
 
+export GOPATH
 export CGO_CFLAGS=$(CGOCFLAGS)
 export CGO_LDFLAGS=$(CGOLDFLAGS)
 
 build:
-	$(GO) env
-	$(GO) build
+	$(AM_V_at)$(GO) build $(top_srcdir)/bindings/go/*.go
```

Comment:
where would output files be written?

Reply:
To the `GOPATH=$(abs_top_builddir)/bindings/go/build-go`

Reply:
ok, maybe no need build-go subfolder, since it's already abs_top_builddir?

Reply:
`build-*` excluded from git, just to not accidentally commit build stuff.

Reply:
we can add binary file patterns to git ignore or use out-of-source build.. IMO no need to create another nesting level; we don't do it with C sources for example